doc/usecases/kdb/UC_load_config.md

```diff
@@ -0,0 +1,31 @@
+# Use Case: Loading application configuration
+
+## Summary
+
+- **Title:** Loading application configuration
+- **Scope:** `libelektra-kdb`
+- **Level:** Developer Goal
+- **Actors:** Application, Elektra
+- **Brief:** Application loads configuration from KDB
+
+## Scenarios
+
+- **Precondition:** -
+- **Main success scenario:**
+  - Application requests current configuration from KDB
+  - Elektra loads from backends
+  - Elektra returns configuration
+- **Alternative scenario:**
+  - Application requests current configuration from KDB
+  - Elektra detects configuration hasn't changed and is already present in memory (of current instance)
+  - Elektra returns previously loaded configuration
+- **Error scenario:**
+  - Application requests current configuration from KDB
```

Comment:
```suggestion
```

Reply:
Why? If the Elektra is a library not a server/daemon, if there is no request/call from an application nothing at all happens.

Reply:
I simply don't think it needs to be repeated in the Error scenario.